RELEASE.md

```diff
@@ -20,44 +20,36 @@ Below is what a typical release week may look like:
 1. Enter the name of the release. Example: `1.2.0` (DO NOT use the v prefix like v1.2.0)
 1. Specify the branch to use for the new release. It's main for all major releases. For a bugfix release, you'll select a different branch.
 1. Click on the `Run workflow` button.
 1. Note: `Run workflow` takes approximately 2-3 minutes.
-1. Close the milestone for the respective release, make sure that all tasks within the milestone are completed / updated before closing. https://github.com/containers/podman-desktop-extension-ai-lab/milestones
+1. Make sure that all tasks for the respective release milestone are completed / updated, then close it. https://github.com/containers/podman-desktop-extension-ai-lab/milestones
 1. If not already created, click on `New Milestone` and create a new milestone for the NEXT release.
 1. Check that https://github.com/containers/podman-desktop-extension-ai-lab/actions/workflows/release.yaml has been completed.
-1. There should be an automated PR that has been created. This will be automatically merged in after all tests have been ran (takes 5-10 minutes). The title looks like `chore: 📢 Bump version to 1.3.0`. Rerun workflow manually if some of e2e tests are failing.
-1. Above PR MUST be merged before continuing with the steps.
-1. Edit the new release https://github.com/containers/podman-desktop-extension-ai-lab/releases/edit/v1.2.0
-1. Select previous tag (v1.1.0) and click on `Generate release notes` and the click on `Update release`
+1. Ensure the image has been successfully published to https://github.com/containers/podman-desktop-extension-ai-lab/pkgs/container/podman-desktop-extension-ai-lab.
+1. There should be an automated PR that has been created. The title looks like `chore: 📢 Bump version to 1.3.0`. Rerun workflow manually if some of e2e tests are failing.
+1. Wait for the PR above to be approved and merged before continuing with the steps.
+1. Edit the new release https://github.com/containers/podman-desktop-extension-ai-lab/releases/edit/v1.2.0.
+1. Select previous tag (v1.1.0) and click on `Generate release notes` and then click on `Update release`.
 
-## Test release before it is rolling out.
+## Release testing
 
-The release is a pre-release, it means it is not yet the latest version, so no clients will automatically update to this version.
+At this stage the release is not in the catalog, so clients will not automatically update to this version.
 
-It allows QE (and everyone else) to test the release before they it will go live on the catalog.
+This allows QE (and everyone else) to test the release before it goes live.
 
+❌ All severe bugs and regressions are investigated and discussed. If we agree any should block the release, we need to fix the bugs and do a respin of the release with a new .z release like 1.2.1 instead of 1.2.0.
 
-## Next phase
+- Create a branch if it does not exist. For example 1.2.1 if 1.2.0 failed. Then, cherry-pick bugfixes into that branch.
```

Comment:
We use 1.2.x on Podman Desktop see https://github.com/containers/podman-desktop/blob/56e136426096a6685f6fb5fb509ba2bd2cb2f916/RELEASE.md?plain=1#L54